test/src/kbasesearchengine/test/main/SearchMethodsTest.java

```diff
@@ -420,9 +422,113 @@ private SearchObjectsOutput searchObjects(
 
         final SearchObjectsInput input = new SearchObjectsInput()
                 .withPostProcessing(pp)
-                .withMatchFilter(new MatchFilter().withFullTextInAll("test"))
+                .withMatchFilter(new MatchFilter().withFullTextInAll(query))
                 .withAccessFilter(new AccessFilter());
 
         return  sm.searchObjects(input, "auser");
     }
-}
+
+    @Test
```

Comment:
Doesn't there need to be a case where the type is null?

Reply:
yes damn